Update IAB Decision process (#104)
ietf/wiki.ietf.org: group/iab/IAB_Decisions.md

## Normal Process

The IAB chair (or executive director on the instructions of the chair) will 
typically follow this process to conduct a vote:

1. Announce the topic or question on which to vote, along with the
   schedule for the vote, to start a period of discussion and consideration
   by the IAB.

2. Ensure that all IAB members have an opportunity to discuss the topic or
   question in some meeting of the IAB.

3. Formally start the vote, following the discussion(s).

4. Conclude the vote once votes from all IAB members have been collected,
   or once 7 days have elapsed and there are sufficient votes received
   to meet the requirements in RFC 2850.

Detailed guidelines for each of these steps are provided below.

### Announcing votes

Announcements of votes are sent over email, but can also be provided over

> other channels. These announcements will be sent by the IAB or the executive director.

Reviewer: ```suggestion
other channels. These announcements will be sent by the IAB chair or the executive director.
```

Announcements should be as specific as possible about the question on which
the vote will be taken, and background material should be provided or already be
available to the IAB.

Announcements can optionally indicate that ballots will be secret.

Announcements should include the timeframe for the vote, including the
opportunities for discussion (see below) and when the vote is expected to start.

### Discussion prior to voting

IAB members should have the opportunity to discuss topics before taking a vote.

> This discussion can occur in both asynchronous (email, messages) and synchronous

Reviewer: ```suggestion
This discussion can occur in both asynchronous (email, slack messages) and synchronous
```

(teleconferences, in-person meetings) forms.

Once a vote is announced, the period for asynchronous discussion starts immediately.
This discsssion generally takes place on a private email list amongst voting
members.

The discussion period also should include opportunities for each voting IAB member
to have synchronous discussions with other voting members. Depending on the exact
meeting schedule of the IAB, this requirement can be satisfied in different ways.

> For example, an in-person meeting at an IETF meeting or an IAB teleconference where

Reviewer: ```suggestion
For example, an in-person meeting at an IETF meeting/retreat or an IAB teleconference where
```

all IAB members are expected to attend satisfies this requirement in a single
discussion. Alternatively, this can be satisfied by having multiple discussions that
occur at different times to accomodate IAB members in different timezones.
Note that lack of attendence by IAB members due to vacations or choosing not to
attend a meeting will not delay being able to hold the vote, as long as meeting
opportunities were provided.

Any meeting on which a vote is discussed needs to either be a regularly scheduled
meeting, or be an extraordinary meeting scheduled at least a week in advance.

Discussions in meetings need to follow a regular procedure for adding agenda items
to the meeting. Voting discussions should be added to a meeting agenda at least
48 hours prior to the meeting, or earlier depending on the current meeting practice
of the IAB.

### Conducting votes

Once the discussion opportunities (described above) have been provided, a vote
can formally start. This can occur during a teleconference or in-person meeting
directly after discussion, but unless all IAB members are present, will always

> also be started virtually.

Reviewer: ```suggestion
also be continued virtually.
```

Reviewer: Suggestion to make it clear that the vote can be started in the IAB meeting following the discussion, and continue on via email for missing votes.

Voting typically occurs on a private email list that includes all voting members,
but may be conducted using other virtual tools.

### Concluding votes

A vote will typically stay open until a position has been recorded for every
single voting member, even after a majority has been reached. This ensures
every voice can be heard. Note that it is reasonable for any voting member to
request a few days to further review available information before voting.

A vote can conclude in either of the following conditions:

- All voting members have submitted their votes
- Seven days have elapsed since the vote was started, and the requirements
  from RFC 2850 have been met (specifically, that seven full members concur
  and there are no more than two dissents)

If these conditions cannot be met within seven days, the IAB chair can
choose to either leave the vote open, or cancel the vote.

## Alternative Process

The IAB can use alternative voting processes. In order to do that, the IAB
chair must announce the alternative voting rules to the IAB. This
announcement should usually occur at least one week before the voting takes

> place. Per RFC 2950, if more than two voting member of the IAB objects to the alternative voting rules,

Reviewer: ```suggestion
place. Per RFC 2850, if more than two voting members of the IAB object to the alternative voting rules,
```

the vote reverts to the Normal Process.

Unanimous results should be strongly preferred in votes using an alternative
process. Particularly, if the vote is run on a shortened timeline (such as

> having less than a week of advanced notice), unanimity should be required.

Reviewer: In the first sentence, it says preferred, but in the 2nd, it says required. Can this be tightened up?

Author: The distinction for me was that if the alternative process is done to be faster, it should require unanimity. However, some hypothetical alternative process could also be a longer process, and may be OK to still have non-unanimity.

Author: ```suggestion
having less than a week of advanced notice), unanimity from those who respond
should be required.
```

### Discussion prior to voting

IAB members should have the opportunity to discuss topics before taking a vote.
This discussion can occur in both asynchronous (email, slack messages) and synchronous
(teleconferences, in-person meetings) forms.

Once a vote is announced, the period for asynchronous discussion starts immediately.
This discsssion generally takes place on a private email list amongst voting
members.

The discussion period also should include opportunities for each voting IAB member
to have synchronous discussions with other voting members. Depending on the exact
meeting schedule of the IAB, this requirement can be satisfied in different ways.
For example, an in-person meeting at an IETF meeting/retreat or an IAB teleconference where
all IAB members are expected to attend satisfies this requirement in a single
discussion. Alternatively, this can be satisfied by having multiple discussions that
occur at different times to accomodate IAB members in different timezones.
Note that lack of attendence by IAB members due to vacations or choosing not to
attend a meeting will not delay being able to hold the vote, as long as meeting
opportunities were provided.

> Any meeting on which a vote is discussed needs to either be a regularly scheduled
> meeting, or be an extraordinary meeting scheduled at least a week in advance.

Author: ```suggestion
Any meeting on which a vote is discussed needs to either be a regularly scheduled
meeting that is on the calendar well ahead of time, or be an extraordinary meeting
scheduled at least a week in advance.
```

also be started virtually.

Voting typically occurs on a private email list that includes all voting members,
but may be conducted using other virtual tools.

### Concluding votes

A vote will typically stay open until a position has been recorded for every
single voting member, even after a majority has been reached. This ensures
every voice can be heard. Note that it is reasonable for any voting member to
request a few days to further review available information before voting.

A vote can conclude in either of the following conditions:

- All voting members have submitted their votes
- Seven days have elapsed since the vote was started, and the requirements
  from RFC 2850 have been met (specifically, that seven full members concur
  and there are no more than two dissents)

If these conditions cannot be met within seven days, the IAB chair can
choose to either leave the vote open, or cancel the vote.

## Alternative Process

The IAB can use alternative voting processes. In order to do that, the IAB
chair must announce the alternative voting rules to the IAB. This
announcement should usually occur at least one week before the voting takes

> place. If any voting member of the IAB objects to the alternative voting rules,
> the resolution of the conflict will be addressed following usual RFC 2850
> guidelines; i.e., if more than two voting members object to the alternative
> voting process, the vote reverts to the Normal Process.

Reviewer 2: ```suggestion
place. Per RFC 2950, if more than two voting member of the IAB objects to the alternative voting rules,
the vote reverts to the Normal Process.
```
